openwisp_controller/config/tests/test_device.py

```diff
@@ -69,6 +69,14 @@ def test_mac_address_validator(self):
             else:
                 self.fail('ValidationError not raised for "{0}"'.format(mac_address))
 
+    def test_device_mac_address_uppercase_from_clean_method(self):
```

Comment:
Let's keep the test names as short as possible.

```suggestion
    def test_device_mac_address_uppercase(self):
```